app/controllers/stories_controller.rb

```diff
@@ -60,6 +62,7 @@ def story_params
 
   def load_story
```

Comment:
```suggestion
  def find_story
```
nên để tên method là find_story